### PR TITLE
Allows to render an array of anchor tags in DetailInfoField

### PR DESCRIPTION
**Pull request by ridoo**
(Needed UI changes for https://github.com/GeoNode/geonode/pull/10367 to display multiple users in a `DetailInfoField`)

`DetailInfoField`s are already is capable to render  an array of children in case context provides an array of values. The field type `link` has been extended  to render `href` from a value object (provided by `PluginContextUtil`). Beforehand, only the `href` from field config was used which is only suitable for a single value IMO. Now, a field of type `link` which misses the `href` attribute the actual rendering uses the data provided from the context.

This change is a suggestion, and I ask kindly for a quick review, if this goes in the right direction. For sure, this can be optimized and may be used for other field types as well.

### CODE REVIEW

**Comment by giohappy** [comment]
@ridoo can you please take a look at the conflicts?

**Comment by ridoo** [comment]
@giohappy I have to take a deeper look at the resolved merge as the POC does not show up in the DetailsList. I give you a ping, once it works again

**Comment by ridoo** [comment]
@giohappy I fixed it. Appeared that field containing an array as value were filtered out (the regions field did not show up also which contained one region in my case). It should work now

File: geonode_mapstore_client/client/js/utils/PluginsContextUtils.js [hunk 1]
```diff
@@ -14,12 +14,33 @@ import {
...
+function getUserResourceName(user) {
+    return user?.first_name !== '' && user?.last_name !== ''
+        ? `${user?.first_name} ${user?.last_name}`
+        : user?.username;
+}
+
+function userListToCsv(users = []) {
+    if (!users) {
+        return "";
+    }
```
Comment (allyoucanmap) [inline geonode_mapstore_client/client/js/utils/PluginsContextUtils.js:26]: Is it possible this case? I'm seeing the the argument users is providing a default empty array so probably this return with an empty string is not really needed.
Maybe is needed if users argument is `null`, `""` or `false` but not sure what the resources endpoint could return for `poc` field

Comment (ridoo) [reply geonode_mapstore_client/client/js/utils/PluginsContextUtils.js:26]: You're right. I guess, this `return` is an artifact from a previous implementation version which did not return an array. I would suggest to return `[]` in such cases. What do you think?

Comment (allyoucanmap) [reply geonode_mapstore_client/client/js/utils/PluginsContextUtils.js]: @ridoo yes, I think the `[]` is fine, thanks

**Review by allyoucanmap — APPROVED** [review]

**Pull request merged**